src/shogun2-core/shogun2-service/src/main/java/de/terrestris/shogun2/service/UserService.java

```diff
@@ -38,6 +48,20 @@ public User findByAccountName(String accountName) {
 	}
 
 	/**
+	 *
+	 * @param email
+	 * @return
+	 */
+	public User findByEmail(String email) {
+
+		SimpleExpression eqEmail = Restrictions.eq("email",
+				email);
+		User user = dao.findByUniqueCriteria(eqEmail);
```

Comment:
Do we already have a unique constraint against `email`? I *think* we may want one.